core/src/main/scala/io/finch/endpoint/multipart.scala

```diff
@@ -38,12 +39,22 @@ private abstract class Multipart[A, B](name: String) extends Endpoint[B] {
 private object Multipart {
   trait Required[A] { _: Multipart[A, A] =>
     protected def missing(name: String): Rerunnable[Output[A]] = Rs.paramNotPresent(name)
-    protected def present(a: A): Rerunnable[Output[A]] = Rs.payload(a)
+    protected def present(a: NonEmptyList[A]): Rerunnable[Output[A]] = Rs.payload(a.head)
```

Comment:
This may fail at runtime with `Seq.empty.head`. I think we should somehow distinguish empty/non-empty cases earlier.

Reply:
Oh please ignore me - that should work just fine given `a` is non-empty!